docs/modules/ROOT/pages/tutorials/tutorial.adoc

```diff
@@ -310,20 +315,38 @@ $ cp -R /data/wordpress-pvc/* .
 [start=3]
 . Since it is a PHP application, nothing else is needed for this pod.
 
-In the case of the MariaDB pod, the application must be restarted after restoring the files:
+=== Restoring the database dump
+
+In the case of the MariaDB pod, this backup annotation in the MariaDB deployment instructed K8up to create a database dump as text and store it in MinIO:
+
+[source,yaml]
+....
+spec:
+  template:
+    metadata:
+      annotations:
+        k8up.syn.tools/backupcommand: appuio.ch/backupcommand: /bin/bash -c 'mysqldump -uroot -p"${MARIADB_ROOT_PASSWORD}" --all-databases'
+....
+
+Thus the restoration procedure is different. Instead of restoring files to a PVC, we can instead connect to the running database in the pod and restore directly from the dump.
```

Comment:
Following the [style guide](https://syn.tools/syn/references/style-guide.html), you should make 1 line per sentence
```suggestion
Thus the restoration procedure is different. 
Instead of restoring files to a PVC, we can instead connect to the running database in the pod and restore directly from the dump.
```